What's new article (#28431)
dotnet/docs: docs/whats-new/dotnet-docs-2022-02-01.md

---

> title: ".NET docs: What's new for February 1, 2022 - February 28, 2022"

Reviewer: I still haven't made this code change:

```suggestion
title: ".NET docs: What's new for February 2022"
```

> description: "What's new in the .NET docs for February 1, 2022 - February 28, 2022."

Reviewer: same:

```suggestion
description: "What's new in the .NET docs for February 2022."
```

ms.date: 03/01/2022
---

> # .NET docs: What's new for February 1, 2022 - February 28, 2022

Reviewer: same:

```suggestion
# .NET docs: What's new for February 2022
```

> Welcome to what's new in the .NET docs from February 1, 2022 through February 28, 2022. This article lists some of the major changes to docs during this period.

Reviewer 2: ```suggestion
Welcome to what's new in the .NET docs for February 2022. This article lists some of the major changes to docs during this period.
```

> 

Reviewer 3: ```suggestion

## Microsoft Orleans

Orleans is a cross-platform framework for building robust, scalable distributed applications. We're pleased to announce that all of the official [Microsoft Orleans](/dotnet/orleans) content has been ported and is now available on the .NET docs.

```

- [How to install the ML.NET Command-Line Interface (CLI) tool](../machine-learning/how-to-guides/install-ml-net-cli.md) - Update ML.NET install doc

## Community contributors

The following people contributed to the .NET docs during this period. Thank you! Learn how to contribute by following the links under "Get involved" in the [what's new landing page](index.yml).

- [guardrex](https://github.com/guardrex) - Luke Latham (4)
- [smholvoet](https://github.com/smholvoet) - Sander Holvoet (3)
- [ardalis](https://github.com/ardalis) - Steve Smith (2)
- [GitHubPang](https://github.com/GitHubPang) (2)
- [Youssef1313](https://github.com/Youssef1313) - Youssef Victor (2)
- [zspitz](https://github.com/zspitz) - Zev Spitz (2)
- [bangjiehan](https://github.com/bangjiehan) (1)
- [cartermp](https://github.com/cartermp) - Phillip Carter (1)
- [ciphercoder](https://github.com/ciphercoder) - Irfan Akram  (1)
- [cjessamy](https://github.com/cjessamy) - Christopher C. Jessamy (1)
- [colejohnson66](https://github.com/colejohnson66) - Cole Tobin (1)
- [dipakdotyadav](https://github.com/dipakdotyadav) - Dipak (1)
- [eatdrinksleepcode](https://github.com/eatdrinksleepcode) - David Nelson (1)

> - [github-actions](https://github.com/github-actions) -  (1)

Reviewer: This bot should be removed:

```suggestion
```
